Lib/test/test_dictviews.py

```diff
@@ -158,6 +158,12 @@ def intersection(self, other):
         self.assertTrue(de.keys().isdisjoint(de.keys()))
         self.assertTrue(de.keys().isdisjoint([1]))
 
+    def test_keys_set_operations_with_iterator(self):
```

Comment:
Maybe add tests for `.items()`?

Reply:
Oh, thanks for the fast response!
I 've added the test you suggested.
Thanks a lot

Reply:
I suggest to move it after `test_items_set_operations`. And rename, it is not only about keys.

Reply:
I rename it into `test_set_operations_with_iterator`. Is it okay?